solr/jmx-tlog-solr7plus.yml

```diff
@@ -0,0 +1,180 @@
+type: jmx
+require:
+  - className: com.sematext.spm.client.solr.SolrVersionCheck
+    values: 7-*
+
+observation:
+  - name: tlog_remainingBytes_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=replay,name=remaining.bytes
+
+    metric:
+      - name: tlog.replay.remaining.bytes
+        source: Value
+        type: long_gauge
+        label: tlog replay remaining bytes
+        description: remaining bytes to replay from the transaction log
+        unit: bytes
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: tlog_remainingLogs_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=replay,name=remaining.logs
+
+    metric:
+      - name: tlog.replay.remaining.logs
+        source: Value
+        type: gauge
+        label: tlog replay remaining logs
+        description: remaining transaction logs to replay
+        unit: logs
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: tlog_replay_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=replay,name=ops
+
+    metric:
+      - name: tlog.replay.ops
+        source: Count
+        type: long_gauge
```

Comment:
Perhaps this is a counter metric, not a gauge?

Reply:
counter. changed